File: app/src/main/java/at/shockbytes/weekly/ui/fragment/SettingsFragment.java
package at.shockbytes.weekly.ui.fragment;

import android.os.Bundle;
import android.preference.PreferenceFragment;

import at.shockbytes.weekly.R;

public class SettingsFragment extends PreferenceFragment {

	@Override
	public void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		//Load preferences from the file
		addPreferencesFromResource(R.xml.prefs);
	}
}

File: app/src/main/java/at/shockbytes/weekly/ui/fragment/SettingsFragment.kt
package at.shockbytes.weekly.ui.fragment

import android.os.Bundle
import android.preference.PreferenceFragment

import at.shockbytes.weekly.R

class SettingsFragment : PreferenceFragment() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        addPreferencesFromResource(R.xml.prefs)
    }
}
